blockexplorer/src/app/components/ValidatorsMap/CircleMarker.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { Marker } from 'react-simple-maps';
+import './styles/CircleMarker.css';
+import Typography from '@mui/material/Typography';
+import Popper, { PopperPlacementType } from '@mui/material/Popper';
+import Fade from '@mui/material/Fade';
+import Paper from '@mui/material/Paper';
+
+const CircleMarker = ({
+  country,
+  lng,
+  lat,
+  rValue,
+  city,
+  sValue,
+  nodes
+}: any) => {
+
+  let cityNodes = nodes;
+
+  const [changeColor, setChangeColor] = useState(false);
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
```

Comment:
I think we can use useRef for storing and manipulating html elements

source: [useRef hook](https://reactjs.org/docs/hooks-reference.html#useref)

Reply:
I changed the useState hook to useRef in the CircleMarker